### DISCUSSION TITLE
Create lessons with the bulk video uploader

### DISCUSSION THREAD

**Discussion by Creeland** [eggheadio/egghead-next#923] [Pitches]
The bulk video uploader #916 currently uploads videos to s3 and returns a video URL. Our next step is to create a list when videos are uploaded in preparation for the final submission where lessons get created. The items in this list will have editable fields for the title and summary. The rest of the data can be filled in on Sanity.

I think Sanity-first is the way to go for lesson data. Lesson resources with the lesson id, external type, slug, summary, and title could be generated in Sanity when lessons are created. Then we wouldn’t have to do any client-side searching for instructors and tags when creating lessons

**Comment by joelhooks** [comment]
This is a subset of https://www.notion.so/egghead/p-egghead-next-course-authoring-63f9a16cc73f4963b5c951a0bf6994f5 and "zoomed in" to the Upload Component

![Image 2021-10-19 at 11 32 05 AM](https://user-images.githubusercontent.com/86834/137969970-c5f61727-606a-45ab-a37b-d2c8700bab71.jpg)

**Comment by joelhooks** [comment]
Here's a miro with some additional info: https://miro.com/app/board/o9J_lTD_OV0=/?invite_link_id=412985250270

<img width="1573" alt="Screen Shot 2021-10-19 at 11 50 37 AM" src="https://user-images.githubusercontent.com/86834/137972671-697c0f10-e126-4d79-8d07-b2146ae4d934.png">
